home.html

```diff
@@ -0,0 +1,94 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="stylesheet.css" rel="stylesheet">
```

Comment:
- [ ] Good job so far 👏 Currently, your styles are not applied 😢 this is how it looks 👇 
![image](https://user-images.githubusercontent.com/90222110/205444880-8aacadf3-1bfd-4b13-9ee0-683ef4f2ad8b.png)

I have seen that you have referenced the wrong stylesheet. Please add the right one 👍 
HINT:  please reference the `home.css`